src/components/Inventory.js

```diff
@@ -22,13 +22,20 @@ export default class Inventory extends React.Component {
     this.props.onToggleProperties(property);
   }
 
+  selectInventoryBlock() {
+    this.props.onSelectInventoryBlock(this.props.data.get('Id'));
+  }
+
   render() {
     return (
       <div className='inventory--MAIN'>
-        <InputField value={this.props.data.Name} callBack={() => this.changeBlockName()} ref='blockName'/>
-        {this.props.data.Properties.filter((property) => property.Type === 'Boolean').map((property) =>
-          <Checkbox checked={property.Value} callBack={() => this.toggleProperty(property.Name)} label={labelList[property.Name]} name={property.Name} key={property.Name} />
-        )}
+        <Checkbox checked={false} callBack={() => this.selectInventoryBlock()} label={false} name={this.props.data.get('Name')} key={this.props.data.get('Name')} />
```

Comment:
Oh snap -- I still owe you a mockup on what multi-select is supposed to do :-O  I'll get that done once I'm done with the PR reviews.